TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -587,10 +587,23 @@ private void RefreshCustomSymbols()
         private void ProgramClassChanged(object cUnit, ProgramClassEvent programEvent)
         {
             var compilationUnit = cUnit as CompilationUnit;
-            var fileUri = OpenedDocumentContext.Keys.FirstOrDefault(k => compilationUnit != null && k.LocalPath.Contains(compilationUnit.TextSourceInfo.Name));
+            var fileUri = OpenedDocumentContext.Keys.FirstOrDefault(k =>
+                compilationUnit != null && k.LocalPath.Contains(compilationUnit.TextSourceInfo.Name));
 
-            var diags = compilationUnit?.AllDiagnostics().Take(TypeCobolConfiguration.MaximumDiagnostics == 0 ? 100 : TypeCobolConfiguration.MaximumDiagnostics);
-            DiagnosticsEvent(fileUri, new DiagnosticEvent() { Diagnostics = diags});
+            var test = compilationUnit?.AllDiagnostics().GroupBy(d => d.Info.Severity);
```

Comment:
test ?
It's also a duplicate line of 
` var severityGroups = compilationUnit?.AllDiagnostics().GroupBy(d => d.Info.Severity);`

Reply:
You are right, I forgot to remove that line before commit